### ISSUE TITLE
any updates on the approval process?

### ISSUE THREAD

**Issue by lano1106** [hhanh00/zcash-ledger#3]
I have zec funds stuck because I made a deshielding transaction that apparently is not supported by ledger live:

https://support.ledger.com/hc/en-us/articles/7497812374941-Zcash-transaction-issues-in-Ledger-Live

I have been looking for a solution that would allow me to access the funds without exporting the private keys out of ledger key...

This project appears to be the option out there...

If I use it will the ywallet be able to access the deshielded funds?

**Comment by hhanh00** [comment]
Ledger and ZF have decided to go with Zondax. https://github.com/Zondax/ledger-zcash

**Comment by lano1106** [comment]
too bad... I really like your ywallet

I did check quickly Zondax and I am not sure if they have a Linux desktop version of their wallet...

BTW, here is a feature suggestion for ywallet... Maybe add a checkbox if the user is about to send coins to a ledger wallet... If it is, perform the deshielding locally (ie break the transaction in 2. Deshield locally. Then send to the ledger destination from a t address to not end up having your coins hostage in the ledger wallet...

Ledger star starts to shine less and less... Zec support has been broken for almost 2 years and they have not cleaned up their mess... (Since version 5, I think...)

**Comment by hhanh00** [comment]
That is unfortunately not possible because there is no way to tell that a taddr belongs to Ledger or not.

**Comment by lano1106** [comment]
correct... but the user knows and could inform the wallet by clicking on a checkbox for that purpose...

**Comment by hhanh00** [comment]
The user can do t2t if he knows Ledger is broken. But hopefully, they will fix it soon.
